template-transformer/spec/handler_spec.lua

```diff
@@ -218,10 +220,24 @@ describe("Test TemplateTransformerHandler body_filter", function()
     _G.ngx.arg = {'{ "key" : "value" }', true}
     TemplateTransformerHandler:body_filter(config)
     assert.equal(config.response_template, ngx.arg[1])
+    assert.equal(200, ngx.status)
```

Comment:
I was considering extracting out the 'when' portion of these tests into a nested describe block with one it block for each assertion...not sure on your thoughts.

e.g.
```
describe('when body is full read and...', function () 
  setup(function () 
     _G.ngx.arg = {'{ "key" : "value" }', true}
     TemplateTransformerHandler:body_filter(config
  end)

  it('should set first nxg arg to template' ... )
  it('should not update the ngx status' ... )

end)
```